libs/client/src/lib.rs

```diff
@@ -1647,6 +1673,7 @@ impl KanidmClient {
             .await
     }
 
+    // TODO: add test coverage
```

Comment:
I thought we did have coverage of this?

Reply:
Coverage doesn't find it in testing and I can't find references to it in the code? 🤔

Reply:
Maybe it's not tested then 🤔

Reply:
Future-us problem. 😄